src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,16 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-#define ONESHOT125_TIMER_MHZ 4
-#define ONESHOT42_TIMER_MHZ 12
```

Comment:
This line should be added back for oneshot42. Shift by one right need to be removed for oneshot output function.